lib/Doctrine/Migrations/Tools/Console/Command/DiffCommand.php

```diff
@@ -8,6 +8,8 @@
 use Doctrine\Migrations\Provider\OrmSchemaProvider;
 use Doctrine\Migrations\Provider\SchemaProviderInterface;
 use Doctrine\Migrations\Tools\Console\Exception\InvalidOptionUsage;
+use const FILTER_VALIDATE_BOOLEAN;
```

Comment:
This is the wrong place, const and function imports are below classes. Running `vendor/bin/phpcs` should report this so you don't have to wait for travis-ci.